### DISCUSSION TITLE
Possibilities of looking into Scaling the RuneLite client smaller

### DISCUSSION THREAD

**Discussion by WaFFlzz** [runelite/runelite#16548] [Ideas]
Hey! I am not a coder and could not contribute to this idea or feature in probably any useful way, but Myself and many friends and fellow multi-loggers, could most definitely make use of a smaller than normal runelite window. Being able to make the window half of the size it is now, or even just a third smaller, would be very nice, because then we could fit 3 clients side by side or possibly up and down on a grid without any clients overlapping, while playing on a 1080p monitor. 

This is just something that has always been on my mind and would be extremely nice for the ever growing amount of us, that play on several accounts at once. The current minimum client size is very inconsiderate for 1080p gamers.

I recently asked this same question in the RuneLite discord and I guess the game crashes if you make the client size too small, but how small is too small? What I am asking for, is any amount of decrease in size, anything at all would be very helpful and greatly appreciated! I have also searched for similar discussions on this site and have found one post from about a month ago that has not had any responses, so many this post will bring more exposure to the idea.

If you need my help I will be watching this site for the next few days, but I couldn't be of much help. 

Thanks!

**Comment by daleloncosky** [comment]
Isn't this something you can do by adjusting the resolution in the Runlite settings?

You can change the res to 1280x720

**Reply by WaFFlzz** [reply]
This is the smallest game size you an achieve as far as I know. I have tried a few things to try to get it smaller and cannot figure anything out that works.

https://i.imgur.com/q4ANP99.png
